### ISSUE TITLE
Use of word Kaavio in various places

### ISSUE THREAD

**Issue by jacobwindsor** [bridgedb/bridgedbjs#6]
In the Typescript branch, there is the use of the word Kaavio in various places. Am I wrong in thinking that this project is separate from Kaavio and would, in fact, be used in PVJS? See [this file name](https://github.com/bridgedb/bridgedbjs/blob/typescript/src/ui/kaavio.css), [this line](https://github.com/bridgedb/bridgedbjs/blob/typescript/src/ui/index.ts#L145), and there are a couple of others I think.

Either way, I think it is confusing to have the names Kaavio or PVJS used anywhere since this library could be used independently from either.

**Comment by jacobwindsor** [comment]
@ariutta is the UI directory meant to be moved to PVJS itself?

**Comment by ariutta** [comment]
Yes, bridgedbjs is separate from Kaavio, and any reference to Kaavio should be removed.

But the UI directory is supposed to be part of bridgedbjs. It is simply some UI elements that could be useful for interacting with the BridgeDb webservice, and it could be used entirely independently of pvjs.

**Comment by ariutta** [comment]
Reopening so we can close it when any reference to Kaavio and/or Pvjs is removed.

**Comment by ariutta** [comment]
The TypeScript branch is pretty much ready to release, so if you don't see any other issues with it, you could merge it into master. It's currently in pre-release on npm: https://www.npmjs.com/package/bridgedb

**Comment by jacobwindsor** [comment]
Okay, great. I'll make some time for removing kaavio/pvjs references at some point

**Comment by ariutta** [comment]
To clarify my comment about it being ready for release, the API client is totally ready. The UI components are the only parts I'm not sure about. They are pretty good, but I just haven't tested them fully (cross-browser, etc.).

**Comment by jacobwindsor** [comment]
Today, I got the UI components working in PVJS. It looks nice. However, one thing is that you cannot drag the box around like you can in current WP. I don't know if that's intentional.
